plugin/litedfm.vim

```diff
@@ -27,7 +27,12 @@ endfunction
 
 
 " See if running CLI or GUI Vim
-let s:context = has('gui_running') ? 'gui' : 'cterm'
+" If user has termguicolors enabled, gui vim is chosen regardless.
+if (&termguicolors)
+    let s:context = 'gui'
+else
+    let s:context = has('gui_running') ? 'gui' : 'cterm'
```

Comment:
Could probably collapse this to `let s:context = (has('gui_running') || &termguicolors) ? 'gui' : 'cterm'`, but that's not a big deal.

Reply:
yeah that's a good point, I can open another request for it if you want.